packages/grafana-ui/src/components/GraphNG/utils.test.ts

```diff
@@ -22,35 +21,62 @@ describe('alignDataFrames', () => {
 
       const aligned = alignDataFrames(data);
 
-      expect(aligned?.frame.fields).toEqual([
-        {
-          config: {},
-          state: {},
-          name: 'time',
-          type: FieldType.time,
-          values: new ArrayVector([1000, 2000, 3000, 4000]),
-        },
-        {
-          config: {},
-          state: {
-            displayName: 'temperature A',
-            seriesIndex: 0,
+      expect(aligned?.fields).toMatchInlineSnapshot(`
```

Comment:
I changed the style to inline snapshot -- is there any advantage to the non snapshot style?